ClimsoftVer4/ClimsoftVer4/ucrMetadataPhysicalFeature.Designer.vb

```diff
@@ -0,0 +1,352 @@
+﻿<Global.Microsoft.VisualBasic.CompilerServices.DesignerGenerated()> _
+Partial Class ucrMetadataPhysicalFeature
+    Inherits ClimsoftVer4.ucrTableEntry
+
+    'UserControl overrides dispose to clean up the component list.
+    <System.Diagnostics.DebuggerNonUserCode()> _
+    Protected Overrides Sub Dispose(ByVal disposing As Boolean)
+        Try
+            If disposing AndAlso components IsNot Nothing Then
+                components.Dispose()
+            End If
+        Finally
+            MyBase.Dispose(disposing)
+        End Try
+    End Sub
+
+    'Required by the Windows Form Designer
+    Private components As System.ComponentModel.IContainer
+
+    'NOTE: The following procedure is required by the Windows Form Designer
+    'It can be modified using the Windows Form Designer.  
+    'Do not modify it using the code editor.
+    <System.Diagnostics.DebuggerStepThrough()> _
+    Private Sub InitializeComponent()
+        Me.btnOpenFile = New System.Windows.Forms.Button()
+        Me.lblFeatureImageFile = New System.Windows.Forms.Label()
+        Me.lblFeaturePicture = New System.Windows.Forms.Label()
+        Me.txtFeaturePicture = New System.Windows.Forms.PictureBox()
```

Comment:
@Muthenya this is a picture box. The pretext is currently txt, could you please change it after merging my PR

Reply:
What should I change it to?

Reply:
I have the prefix now as "pb".